docs/js/navigation.js

```diff
@@ -6,7 +6,7 @@ document.write('\
 				<a href="index.html">Home</a>\
 			</li>\
 			<li class="nav-text">\
-				<a href="about.html">About</a>\
+				<a href="about.html">About OpenEXR</a>\
```

Comment:
I removed the "OpenEXR" here so it reads just "About". Can you fix?

Reply:
Fixed!